api-docs/generator.py

```diff
@@ -1,40 +1,120 @@
+#!/usr/bin/env python2.7
 from __future__ import absolute_import
 
-import os
-import zlib
-import json
 import click
-import logging
-import six
-
-from datetime import datetime
-from subprocess import Popen, PIPE, check_output
-from six.moves.urllib.parse import urlparse
+import docker
+import json
+import os
+from sentry.runner.commands.devservices import get_docker_client, get_or_create
+from sentry.conf.server import SENTRY_DEVSERVICES
+from subprocess import Popen
 
 HERE = os.path.abspath(os.path.dirname(__file__))
+OUTPUT_PATH = "/usr/src/output"
 SENTRY_CONFIG = os.environ["SENTRY_CONF"] = os.path.join(HERE, "sentry.conf.py")
 os.environ["SENTRY_SKIP_BACKEND_VALIDATION"] = "1"
 
-# No sentry or django imports before this point
+client = get_docker_client()
+
+# Use a unique network and namespace for our apidocs
+namespace = "apidocs"
+network = get_or_create(client, "network", namespace)
+
+
+# Define our set of containers we want to run
+containers = {
+    "postgres": {
+        "image": SENTRY_DEVSERVICES["postgres"]["image"],
+        "ports": {"5432/tcp": ("127.0.0.1", 5400)},
```

Comment:
@MeredithAnya How come we use `127.0.0.1` here but we switched from this to `localhost` in other parts of the PR?

Reply:
127.0.0.1 is more reliable and safe than localhost. We really shouldn't use localhost anywhere.